src/main/java/uk/gov/ons/census/fwmt/jobservice/consumer/queue/RmReceiver.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.ons.census.fwmt.jobservice.consumer.queue;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.ons.census.fwmt.common.error.GatewayException;
+import uk.gov.ons.census.fwmt.events.component.GatewayEventManager;
+import uk.gov.ons.census.fwmt.jobservice.dto.rm.FieldworkFollowup;
+import uk.gov.ons.census.fwmt.jobservice.service.JobService;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+
+@Slf4j
+@Component
+public class RmReceiver {
+  private final JobService jobService;
+  private final GatewayEventManager gatewayEventManager;
+  private final JAXBContext jaxbContext;
+  private final Unmarshaller unmarshaller;
+  private final ObjectMapper objectMapper;
+
+  public RmReceiver(JobService jobService, GatewayEventManager gatewayEventManager) throws JAXBException {
```

Comment:
By default can you use @Autowired rather than DI via contructor, its how our other projects are done, consistency